lib/contestant.rb

```diff
@@ -0,0 +1,30 @@
+class Contestant
+  attr_reader :first_name, :last_name, :age, :state_of_residence,
+              :spending_money, :out_of_state, :game_interests
+  def initialize(info, out_of_state = false)
+    @first_name = info[:first_name]
+    @last_name = info[:last_name]
```

Comment:
it's fine to capture first and last separately, but since the interaction pattern doesn't really ask us for this, we could just replace these with full_name